src/sync.js

```diff
@@ -36,7 +36,7 @@ const DEBUG = true;
 // beyond that and wedge forever, so we need to track how long we are willing
 // to keep open the connection. This constant is *ADDED* to the timeout= value
 // to determine the max time we're willing to wait.
-const BUFFER_PERIOD_MS = 80 * 1000;
+const BUFFER_PERIOD_MS = 30 * 1000;
```

Comment:
Decreased this since it is now debounced; I think 60 seconds to wait for an onProgressEvent is enough before timing out?